parkhangs-expressapp/models/EventModel.js

```diff
@@ -8,7 +8,7 @@ const Event = new Schema(
         eventDateTime: {type: String, required: true},
         createdDateTime: {type: String, required: true},
         creatorName: {type: String, required: true},
-        creatorId: {type: Number, required: true}
+        creatorId: {type: Number, required: false}
```

Comment:
why is this changed?

Reply:
because I wasn't allowed to update an event without providing a creator id. I can change it back and add a fake one in my controller.